tests/page/selectors-css.spec.ts

```diff
@@ -90,10 +91,10 @@ it('should work for open shadow roots', async ({ page, server }) => {
   expect(await page.$eval(`css=section > div div span:nth-child(2)`, e => e.textContent)).toBe('Hello from root3 #2');
   expect(await page.$(`css=section div div div div`)).toBe(null);
 
-  const root2 = await page.$(`css=div div`);
+  const root2 = (await page.$(`css=div div`))!;
   expect(await root2.$eval(`css=#target`, e => e.textContent)).toBe('Hello from root2');
   expect(await root2.$(`css:light=#target`)).toBe(null);
-  const root2Shadow = await root2.evaluateHandle(r => r.shadowRoot);
+  const root2Shadow = (await root2.evaluateHandle(r => r.shadowRoot) as ElementHandle<ShadowRoot>)!;
```

Comment:
Perhaps the following will also work?

```suggestion
  const root2Shadow = await root2.evaluateHandle(r => r.shadowRoot!);
```